remoting/src/main/java/org/apache/rocketmq/remoting/netty/NettyRemotingClient.java

```diff
@@ -95,7 +96,7 @@ public class NettyRemotingClient extends NettyRemotingAbstract implements Remoti
     private final ConcurrentHashMap<String /* cidr */, Bootstrap> bootstrapMap = new ConcurrentHashMap<>();
     private final ConcurrentMap<String /* addr */, ChannelWrapper> channelTables = new ConcurrentHashMap<>();
 
-    private final Timer timer = new Timer("ClientHouseKeepingService", true);
+    private final HashedWheelTimer timer = new HashedWheelTimer(r -> new Thread(r, "ClientHouseKeepingService"));
```

Comment:
we should  keep it simple .  it is necessary ?

Reply:
HashedWheelTimer is more accurate than java.util.Time.

Reply:
In my opinion, the network module as the basic component, we should try to optimize every detail.
Especially in the stream processing scenario, some details of the performance issues, this can cause some unexpected performance problems.
HashedWheelTimer is more reasonable and better than Timer in terms of performance and implementation.